openra/tests/test_service_docker.py

```diff
@@ -0,0 +1,128 @@
+import datetime
+
+from django.utils import timezone
+from unittest import TestCase
+from unittest.mock import  Mock, MagicMock
+from openra.services.docker import Docker
+
+class TestServiceDocker(TestCase):
+
+    def test_test_docker_will_pass_the_correct_parameters_to_docker_run(self):
```

Comment:
`test_test_`?

Reply:
The name of the function it is testing is called "test_docker". I've updated this now to read `test_that_test_docker...` to make it look less wrong.